benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -874,13 +876,14 @@ void GraphicsBenchmarkApp::ProcessKnobs()
     const bool alphaBlendKnobChanged       = pAlphaBlend->DigestUpdate();
     const bool depthTestWriteKnobChanged   = pDepthTestWrite->DigestUpdate();
     const bool enableSpheresKnobChanged    = pEnableSpheres->DigestUpdate();
+    const bool requireMoreSpheres          = (pSphereInstanceCount->GetValue() > mInitializedSpheres);
 
     const bool enableSpheres = pEnableSpheres->GetValue();
     // If the LOD is empty, assuming we skipped the setup for all sphere resources at start
     // So need to do the intial setup for all resources here
     const bool spheresAreSetUp               = mSpheresAreSetUp;
     const bool updateSphereDescriptors       = spheresAreSetUp && allTexturesTo1x1KnobChanged;
-    const bool setupSphereResourcesAndMeshes = (!spheresAreSetUp) && enableSpheres;
+    const bool setupSphereResourcesAndMeshes = (!spheresAreSetUp || requireMoreSpheres) && enableSpheres;
```

Comment:
I believe this change will cause `SetupSphereResources()` to be called more than once, but that has logic like loading shaders which probably should only be called once.

Reply:
Update.